_data/l10n.yml

```diff
@@ -151,6 +151,7 @@ it:
       sort_by_vitality: Più attivi
       sort_by_alphabetic: Alfabetico
       categories: Categorie
+      itendend_audience: Ambito di applicazione
```

Comment:
```suggestion
      intendend_audience: Ambito di applicazione
```

Reply:
holy crap I had something in my eyes lol